src/conn/conn_dhandle.c

```diff
@@ -313,6 +313,17 @@ __wt_conn_dhandle_close(WT_SESSION_IMPL *session, bool final, bool mark_dead)
     btree = is_btree ? dhandle->handle : NULL;
 
     if (is_btree) {
+        /*
+         * !! Check if we have exclusive access here, else a transaction can start on this btree
+         * after this check !!
```

Comment:
We don't have comments formatted like this in the library. I suggest removing the exclamation points and ending the sentence with a period.

Reply:
Done